package.json

```diff
@@ -8,8 +8,8 @@
   "repository": "github:nextstrain/nextstrain.org",
   "homepage": "https://nextstrain.org",
   "engines": {
-    "node": "^14",
-    "npm": "^6.14"
+    "node": "^18",
```

Comment:
[This error from `npm run build`](https://github.com/nextstrain/nextstrain.org/actions/runs/3137943731/jobs/5096707413#step:6:143):

```
error UNHANDLED REJECTION error:0308010C:digital envelope routines::unsupported
```

was also observed in https://github.com/nextstrain/auspice/pull/1520. Webpack, an Auspice dependency, [is the culprit](https://github.com/webpack/webpack/issues/14532). This will be fixed once that PR is merged, released, and the Auspice version is bumped for this project.